liteflow-labs/liteflow-js
# Remove @nft/templates library

### Project organization

- Dependant on https://github.com/liteflow-labs/liteflow-js/pull/90

### Description

Remove the templates library as this is being merged into the https://github.com/liteflow-labs/marketplace-template/pull/45

### How to test

Build the libraries and link them to the template then check that all pages are working as expected

Note that the failed deployment is normal as the storybook has been removed for templates

### Checklist
